src/generatePythonFiles.sh

```diff
@@ -22,5 +22,11 @@ for path in [ # Search for libcircllhist.so
      pass
 if not C:
    # let dlopen throw it's error
+   print("""
+
+libcircllhist.so was not found on your system.
+Please install libcircllhist from: https://github.com/circonus-labs/libcircllhist/
+
+   """)
    ffi.dlopen("libcircllhist.so")
 EOF
```

Comment:
Also this file seems to have some strange indenting, 2 and **3** spaces! Should be all 4 spaces.